aim/tools/cli/commands/infra.py

```diff
@@ -56,3 +56,32 @@ def create(ctx):
     ctx.obj['apic_manager'].ensure_infra_created_on_apic()
     ctx.obj['apic_manager'].ensure_bgp_pod_policy_created_on_apic()
     ctx.obj['apic_manager'].ensure_opflex_client_cert_validation_disabled()
+
+
+@infra.command(name='tag-list')
```

Comment:
Is that possible to add UTs for these 2 APIs?

Reply:
Will look into that.